settings.gradle.kts

```diff
@@ -11,6 +11,7 @@ include("heap-dump")
 include("client-protocol")
 include("instrumentation-support")
 include("studio-agent")
+include("studio-plugin")
```

Comment:
studio-plugin is an Android Studio plugin, and studio-agent is a java agent that intruments `DefaultGradleConnector` so measure Gradle time. So it's a different thing and I think it's good that they are separated. 

Maybe we could just name studio-agent differently

Reply:
That's not what I meant, sorry. If they have to be different things (an agent vs an IntelliJ plugin), then it's fine to keep them in the projects they are now, separated.

My question was about whether whatever the agent was doing today could also be implemented by an IntelliJ plugin. If so, then I'd favor dropping the agent and moving the code to the plugin to keep things simple. If not, then we need to keep the agent.

Reply:
So I think it's good that we keep the agent since the agent attaches directly to Gradle and measures Gradle time more accurately. The plugin calls Android Studio API that does some extra work, so the time we can measure there is not exactly just the time of the Gradle.

Reply:
👍